core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Path2afp.java

```diff
@@ -2062,7 +2067,8 @@ static boolean computeControlPointBoundingBox(PathIterator2afp<?> iterator,
 			default:
 			}
 		}
-		if (foundOneControlPoint) {
			box.setFromCorners(xmin, ymin, xmax, ymax);
+		if (foundOneControlPoint) {
```

Comment:
Probably an accident here. No change actually.  As for the other issues:
I have looked in the logs fro S1192 issue; most of the time it is either a
blank line missing  after a variable definition or an indentation problem.
So that + sign is the cause  of that problem or I still get my indentation
right? But in IntelliJ IDE, I saw no indentation problem. I set the spacing
 to spaces and not tabs.

On Mon, May 30, 2016 at 8:55 PM, Stéphane Galland <notifications@github.com>
wrote:

> In
> core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/Path2afp.java
> <https://github.com/gallandarakhneorg/afc/pull/66#discussion_r65096448>:
>
> > @@ -2062,7 +2067,8 @@ static boolean computeControlPointBoundingBox(PathIterator2afp<?> iterator,
> >  			default:
> >  			}
> >  		}
> > -		if (foundOneControlPoint) {
> 			box.setFromCorners(xmin, ymin, xmax, ymax);
> > +		if (foundOneControlPoint) {
>
> What is the difference?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gallandarakhneorg/afc/pull/66/files/b7612500ef76145bcb7a31af1f186dad0f7078e7#r65096448>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AR09BMLbGv0EKtl1kFvM4kA_ijmPMclfks5qGyR4gaJpZM4Ipt4W>
> .
>